src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -687,4 +687,109 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="ShowError_ApplySettingFailed" xml:space="preserve">
     <value>Applying setting failed.</value>
   </data>
+  <data name="NuGetUpgrade_Progress_Installing" xml:space="preserve">
+    <value>Installing required packages</value>
+  </data>
+  <data name="NuGetUpgrade_Progress_Uninstalling" xml:space="preserve">
+    <value>Uninstalling packages from packages.config</value>
+  </data>
+  <data name="NuGetUpgrade_WaitMessage" xml:space="preserve">
+    <value>Upgrading Project</value>
+  </data>
+  <data name="NuGetUpgrade_PackageDependencyOf" xml:space="preserve">
+    <value>(dependency of {0})</value>
+  </data>
+  <data name="Text_ConvertPackagesConfigIntroText1" xml:space="preserve">
+    <value>This tool will switch your project from using packages.config to using project.json to maintain your NuGet packages.</value>
```

Comment:
Remove project.json here and update this. It looks like this applies to a lot of the messages here.